feature-group-definitions/backdrop-filter.dist.yml

```diff
@@ -0,0 +1,17 @@
+# Generated from: backdrop-filter.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: backdrop-filter
+description: The `backdrop-filter` CSS property applies graphical effects such as blurring or color shifting to the area behind an element.
+spec: https://drafts.fxtf.org/filter-effects-2/#BackdropFilterProperty
+caniuse: css-backdrop-filter
+status:
+  baseline: false
+  support:
+    chrome: "76"
+    chrome_android: "76"
+    edge: "17"
```

Comment:
Oops, https://caniuse.com/css-backdrop-filter disagrees about this. Will back out to look at separately.